helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestAutoRebalancePartitionLimit.java

```diff
@@ -54,7 +56,13 @@ public void beforeClass() throws Exception {
     _gSetupTool.addCluster(CLUSTER_NAME, true);
 
     _gSetupTool.addResourceToCluster(CLUSTER_NAME, TEST_DB, 100, "OnlineOffline",
-        RebalanceMode.FULL_AUTO + "", 0, 25);
+        RebalanceMode.FULL_AUTO.name(), 0, 25);
+
+    IdealState idealState =
+        _gSetupTool.getClusterManagementTool().getResourceIdealState(CLUSTER_NAME, TEST_DB);
+    idealState.setRebalancerClassName(AutoRebalancer.class.getName());
```

Comment:
add a comment, we need autoRebalancer + autoRebalance strategy. here.

Reply:
Sure. Let me note this is to ensure we are testing AutoRebalancer.